src/main/java/seedu/address/model/ingredient/Ingredient.java

```diff
@@ -10,6 +10,18 @@
  */
 
 public class Ingredient implements Item {
+    public static final String MESSAGE_CONSTRAINTS_INGREDIENTNAME =
+            "Ingredient should only contain alphabets and spaces, and it should not be blank";
+
+    public static final String MESSAGE_CONSTRAINTS_INGREDIENTUNIT =
+            "Unit should be integers only";
```

Comment:
Positive integer?

Reply:
Check last commit: I utilised String.Util to check it is non-zero unsigned integer.